cpp/src/arrow/parquet/parquet-io-test.cc

```diff
@@ -196,13 +290,31 @@ TEST_F(TestParquetIO, SingleColumnInt64ChunkedWrite) {
   ASSERT_TRUE(values->Equals(out));
 }
 
+TEST_F(TestParquetIO, SingleColumnTableInt64ChunkedWrite) {
```

Comment:
you might consider using parameterized tests so you can go through all primitive types concisely